pkg/workloads/console/controller.go

```diff
@@ -221,8 +222,12 @@ func (r *reconciler) Reconcile() (res reconcile.Result, err error) {
 	}
 
 	switch {
-	case r.console.Pending():
-		res = requeueAfterInterval(r.logger, time.Second)
+	case r.console.PreRunning():
```

Comment:
Because PreRunning encapsulates `{Creating, PendingAuthorisation, Pending}`, the behaviour of this case block has now changed: We used to requeue every second if in the `Creating` phase, but now we reconcile after `TTLSecondsBeforeRunning()`.

I think maybe we should flip this, and it's more correct to have the conditional be specific to the `PendingAuthorisation` phase, else we reconcile in a second. Might be more defensible to future code changes?

Reply:
Discussed: We'll try to remove this first conditional, and only requeue after TTLSecondsBeforeRunning